pkg/api/serialization_test.go

```diff
@@ -125,7 +124,7 @@ func TestList(t *testing.T) {
 	roundTripSame(t, item)
 }
 
-var nonRoundTrippableTypes = util.NewStringSet("ThirdPartyResource")
```

Comment:
I guess Brendan added ThirdPartyResource here is because the fuzzer was not there and the test failed. Since we have the fuzzer now we can remove it.